Project.toml

```diff
@@ -16,6 +16,9 @@ GeometryTypes = "4d00f742-c7ba-57c2-abde-4428a4b178cb"
 IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
```

Comment:
Please add a compat line for each new dependency

Reply:
This is required by JuliaRegistrator

Reply:
Done!